ios/connect.go

```diff
@@ -333,3 +362,39 @@ func initializeXpcConnection(h *http.HttpConnection) error {
 
 	return nil
 }
+
+func ConnectUserSpaceTunnel(remoteIp string, port int) (*net.TCPConn, error) {
+	addr, _ := net.ResolveTCPAddr("tcp4", fmt.Sprintf("localhost:%d", DefaultTunnelPort()))
+	conn, err := net.DialTCP("tcp", nil, addr)
+	if err != nil {
+		return nil, fmt.Errorf("ConnectUserSpaceTunnel: failed to dial: %w", err)
+	}
+	_, err = conn.Write(net.ParseIP(remoteIp).To16())
+	portBytes := make([]byte, 4)
+	binary.LittleEndian.PutUint32(portBytes, uint32(port))
+	_, err1 := conn.Write(portBytes)
+	return conn, errors.Join(err, err1)
+}
+
+// defaultHttpApiPort is the port on which we start the HTTP-Server for exposing started tunnels
+// 60-105 is leetspeek for go-ios :-D
+const defaultHttpApiPort = 60105
+
+// DefaultHttpApiPort is the port on which we start the HTTP-Server for exposing started tunnels
+// if GO_IOS_AGENT_PORT is set, we use that port. Otherwise we use the default port 60106.
+// 60-105 is leetspeek for go-ios :-D
+func DefaultHttpApiPort() int {
+	port, err := strconv.Atoi(os.Getenv("GO_IOS_AGENT_PORT"))
+	if err != nil {
+		return defaultHttpApiPort
+	}
+	return port
+}
+
+// DefaultTunnelPort is the port on which you can connect to services on the device.
+// It is used for connecting to the user space TUN device.
+// It will be on DefaultHttpApiPort() + 1
+// Which means 60106 per default
+func DefaultTunnelPort() int {
```

Comment:
if `GO_IOS_AGENT_PORT` can be set, this one should be configurable too

Reply:
This one actually needs to change so every device gets its own port. Right now, all conns go to the same device

Reply:
Ah ok, at the moment we can have one device per agent process? I assume that will change in the future and this here will become a port range?